src/app/modules/convai-checker/perspective-status.component.ts

```diff
@@ -365,16 +357,9 @@ export class PerspectiveStatusComponent implements OnChanges, OnInit, AfterViewI
             }
           }
         }
-        if (this.hideLoadingIconAfterLoadChanged
-            || this.alwaysHideLoadingIconChanged) {
-          if (this.hideLoadingIconAfterLoadChanged) {
-            console.debug('Setting hideLoadingIconAfterLoadChanged to false');
-            this.hideLoadingIconAfterLoadChanged = false;
-          }
-          if (this.alwaysHideLoadingIconChanged) {
-            console.debug('Setting alwaysHideLoadingIconChanged to false');
-            this.alwaysHideLoadingIconChanged = false;
-          }
+        if (this.showFeedbackChanged) {
+          this.showFeedbackChanged = false;
+          console.debug('Setting showFeedbackChanged to false');
```

Comment:
nit: Could we remove the `console.debug`? I also use console logging for debugging but it does tend to clutter the console and I think it might be best to keep the production code free from them.

Reply:
Same thing for the `console.log`s throughout the file.

Reply:
So I made a lot of these console.debugs and not console.logs because that way they can be easily filtered out from the main logs. For a lot of the animation code in this file, debugging can be a huge pain and it's often useful to have them there and not need to re-add logs for everywhere that could become a problem. But I do acknowledge we may not want users to see them. I'm wondering if there's a good way to filter these out for prod builds but can still retain them for our own debugging. Maybe something like this? https://www.codementor.io/@brijmcq/angular-clear-all-of-your-console-logs-in-production-build-with-just-a-few-lines-of-code-cxcw30yqs Let me know what you think.

Reply:
That's totally fair. I'm OK with leaving them as-is for now and creating an Issue to find a nice way to automatically suppress these messages in prod. I think the ideal would be to find some sort of NPM package that strips the JS file of any `console.log`s or `console.debug`s. I got the idea from https://www.codebyamir.com/blog/suppressing-console-log-messages-in-production

Reply:
Okay. I created https://github.com/conversationai/perspectiveapi-authorship-demo/issues/99 so we can remember to handle this later.

Reply:
Okay! I created https://github.com/conversationai/perspectiveapi-authorship-demo/issues/99 so we can remember to fix this later.